scheduler/src/cook/kubernetes/controller.clj

```diff
@@ -307,6 +307,7 @@
                                         ; TODO: This can also legitimately occur in a
                                         ; normal occurrence if someone submits a kill request on a job that fails to launch.
                                         :missing (do
+                                                   ; TODO: We will review these and may downgrade this to an info later.
                                                    (log/warn "In compute cluster" name ", pod" pod-name "in a weird cook expected state:" cook-expected-state-dict "and k8s actual state" k8s-actual-state-dict)
```

Comment:
This should be using the `prepare-for-logging` functions.

Reply:
Ok.